src/TesApi.Web/Management/Clients/TerraLandingZoneApiClient.cs

```diff
@@ -11,7 +11,7 @@ namespace TesApi.Web.Management.Clients
     /// </summary>
     public class TerraLandingZoneApiClient : HttpApiClient
     {
-        private const string LandingZonesApiSegments = @"/api/landingzones/v1/azure";
+        private const string LandingZonesApiSegments = @"/api/landingzones/v1/azure/";
         private const string TokenScope = @"https://management.azure.com/.default";
```

Comment:
this will eventually need to be config because when running in Azure China or Azure Government, it has a different domain for the management plane